[doc] Improve wording on Xprop (#16844)
lowRISC/opentitan: doc/ug/dv_methodology/index.md

Xprop is enabled by default when running simulations for all of our DUTs due to the acceptable level of overhead it adds in terms of wall-clock time (less than 10%).

It's mandatory to enable Xprop when running regression for coverage closure.
Please refer to the simulator documentation to identify and apply the necessary build-time and / or run-time options to enable Xprop.

> On OpenTitan, Xprop is already enabled by default for VCS and Xcelium simulators.

Reviewer: Nit:
```suggestion
In OpenTitan, Xprop is enabled by default for VCS and Xcelium simulators.
```

Author: Fixed.

  endfunction : invalidate_a_channel
```

> The simulator may report that some portions of RTL / DV could not instrumented for X-propagation due to the way they were written.

Reviewer: ```suggestion
The simulator may report that some portions of RTL / DV could not be instrumented for X-propagation due to the way they were written.
```

Author: Fixed.
